getting-started/setup-building.rst

```diff
@@ -414,27 +414,31 @@ OpenSSL which means that you will not be able to build the ``_ssl`` extension.
 One solution is to install these libraries from a third-party package
 manager, like Homebrew_ or MacPorts_, and then add the appropriate paths
 for the header and library files to your ``configure`` command.  For example,
+with **Homebrew**,
 
-with **Homebrew**::
+For Python 3.10 and newer::
 
-    $ brew install pkg-config openssl@1.1 xz gdbm tcl-tk
+    $ brew install pkg-config openssl xz gdbm tcl-tk
```

Comment:
Are we sure this is right?

Reply:
I added this line because the guide currently recommends installing `openssl@1.1`, but for `configure` we see the following:

`--with-openssl=$(brew --prefix openssl)`

Furthermore, for Homebrew `openssl` is an alias for `openssl@3`, not `openssl@1.1`.

I could either revert back the `brew install` command to its original form, but if that's the case, then I believe the best option would be to also use `openssl@1.1` for `--with-openssl` for Python 3.10 and newer to avoid confusion, i.e. `--with-openssl=$(brew --prefix openssl@1.1)`.

Reply:
Same thing for MacPorts, `openssl` is an alias for `openssl3`.

Reply:
Python 3.10 and above require OpenSSL 1.1.1 or newer, so both `openssl@3` and `openssl@1.1` in Homebrew or `openssl3` and `openssl11` in MacPorts work for building Python, I don't know if OpenSSL 3 might work for Python 3.7 through 3.9.

Reply:
FWIW this was changed recently -- have you double checked `git blame` and the discussion in the related PR?

Reply:
What @ezio-melotti said. Please have a look at the recent discussion. Check the closed issues and merged PRs for this repo.

Reply:
@ezio-melotti, I found PR https://github.com/python/devguide/pull/892, and they only modified the package to install via `brew` from `openssl` to `openssl@1.1`, but for `configure` they're still referencing `openssl` (a.k.a. `openssl3`) instead of `openssl@1.1`.

So this is what I'm suggesting

Revert the `brew` command to: 

```sh
$ brew install pkg-config openssl@1.1 xz gdbm tcl-tk
```

And update to `configure` options to:

```sh
$ CFLAGS="-I$(brew --prefix gdbm)/include -I$(brew --prefix xz)/include" \
   LDFLAGS="-L$(brew --prefix gdbm)/lib -I$(brew --prefix xz)/lib" \
   ./configure --with-pydebug \
                      --with-openssl="$(brew --prefix openssl@1.1)"
```

Reply:
Seems like this all works and is approved by Ned, so marking this as resolved